src/GraphQL/Validation/Complexity/ComplexityAnalyzer.cs

```diff
@@ -9,11 +10,11 @@ namespace GraphQL.Validation.Complexity
     public class ComplexityAnalyzer : IComplexityAnalyzer
     {
         /// <inheritdoc/>
-        public void Validate(GraphQLDocument document, ComplexityConfiguration complexityParameters)
+        public void Validate(GraphQLDocument document, ComplexityConfiguration complexityParameters, ISchema? schema = null)
```

Comment:
If this is a breaking change anyway, perhaps we should make `schema` required.

Reply:
It still can work without scheme as before